src/masonite/validation/Validator.py

```diff
@@ -791,6 +791,38 @@ def negated_message(self, attribute):
         return "The {} matches pattern {} .".format(attribute, self.pattern)
 
 
+class postal_code(BaseValidation):
+    def __init__(self, validations, locale, messages={}, raises={}):
+        super().__init__(validations, messages=messages, raises=raises)
+        from .resources.postal_codes import PATTERNS
+
+        try:
+            pattern_dict = PATTERNS.get(locale)
+            if pattern_dict["pattern"] is None:
+                # TODO: how to raise ?
+                print("Postal code validation not implemented for {}".format(locale))
+            else:
+                self.pattern = pattern_dict["pattern"]
+                self.pattern_example = pattern_dict["example"]
+        except KeyError:
+
+            # TODO: how to raise an Error in init of a validation rule ?
```

Comment:
What is this TODO referring to? why can't we raise an error like you are doing? I don't see why we couldn't just raise an error here?

Reply:
I don't know if raising an exception was the good way to handle error in this validation packages. I did not check yet if there were some functions to handle that inside the validation package.
But I am okay with raising an error 👍